2016-04-08.md

```diff
@@ -0,0 +1,37 @@
+We are thrilled to announce that Tessel 2 has shipped to all domestic and international pre-orders.
+
+To our knowledge, this is the first time a distributed, unpaid team of open source contributors has shipped a physical product. This is a major milestone in the open source hardware movement.
+
+Huge thanks to everyone who has contributed so far to the Tessel Project. Whether you're a core contributor, someone who found a typo to fix, or one of our test subjects for the start experience: thank you. We value what you've done.
+
+### "This sounds like a cool project, how can I get involved?"
+
+New contributors are always welcome! Here are a few ways to get involved:
+
+* Read through the [Tessel 2 Technical Overview](https://github.com/tessel/onboarding/blob/master/T2-TECHNICAL-OVERVIEW.md) to get an idea of T2's tech stack, then pick a repo that looks interesting to you. There are plenty of open issues to choose from.
+* Go to the [Tessel forums](https://tessel.io/forums) and help someone solve their problem. Whether it's a bug on our end or theirs, you will probably learn something.
+* Join us on the [Tessel Slack](https://tessel-slack.herokuapp.com/) and ask about contributing to the project - we'll talk to you!
+
+You can read more about how the Tessel Project works as an organization under open governance on the [Tessel Project repo](https://github.com/tessel/project).
+
+### "I just got my Tessel 2, what do I do?"
+
+The best place to start with your Tessel 2 is at [tessel.io/start](//tessel.io/start). You can find tutorials there for everything from blinking lights to tweeting to creating your own local internet.
+
+If you're interested in trying something off the beaten path - electrical fun, advanced hacks, and more, check out [tessel.io/docs](//tessel.io/docs). If you're new to hardware, don't worry! They're intended to be accessible to electrical newbies - and if they're not, please [file an issue](https://github.com/tessel/t2-docs/issues) to let us know.
+
+We're continuing to update the start experience and the Tessel 2 docs, so please check back again!
+
+*"But wait, I don't think mine shipped!" - if this is you, please contact support@tessel.io, and we will figure it out. Please do note that orders from Seeed Studio's page are fulfilled by Seeed directly, so we won't have any information about them.*
```

Comment:
I don't think you need the "so we won't have any information about them". We can forward them on to Seeed to get more information.